test/cli.js

```diff
@@ -303,6 +305,47 @@ describe('CLI', () => {
         });
     });
 
+    it('starts the inspector with --inspect', (done) => {
```

Comment:
It can do that, but there isn't a need to add the extra check, the output should be enough.